.github/workflows/ci.yml

```diff
@@ -241,3 +241,72 @@ jobs:
         uses: docker/build-push-action@v6
         with:
           context: site/redirector/landing
+
+  sw_build_test:
+    name: Build and test software
+    runs-on: ubuntu-22.04-vivado
+    timeout-minutes: 120
+    needs: quick_lint
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Required so we can lint commit messages.
```

Comment:
Do we still need this for sw build & test?

Reply:
The comment is wrong, but we do need this for the bitstream cache to work as well. I'm going to remove this and see if this job does any splicing.